concrete/src/User/Group/Command/AddGroupCommandHandler.php

```diff
@@ -78,6 +78,7 @@ public function __invoke(AddGroupCommand $command)
         if (is_object($node)) {
             GroupNode::add($ng, $node);
         }
+        $ng->rescanGroupPath();
```

Comment:
This change is required, otherwise `group->getGroupPath()` would return NULL in the logging-related functions.